ext/popup.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Save workspaces</title>
+    <link href="https://fonts.googleapis.com/css?family=Pathway+Gothic+One" rel="stylesheet">
```

Comment:
You can inline the font file to make it load instantly. Another alternative is to include the font file as an asset of the extension.